ci-operator/step-registry/openshift-qe/installer/azure/aro/openshift-qe-installer-azure-aro-workflow.yaml

```diff
@@ -0,0 +1,13 @@
+workflow:
```

Comment:
Should we use [this](https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/azure/aro/cucushift-installer-rehearse-azure-aro-workflow.yaml) workflow instead? seems to be the exact same

Reply:
I talked to Mike who used to be  in ARO team. He suggested that we create our own workflow.

Reply:
@qiliRedHat any thoughts on this? This sounds ok to me